src/main/webapp/aboutus.html

```diff
@@ -21,7 +21,7 @@
     <title>About Us</title>
     <link rel="stylesheet" href="/css/main.css">
   </head>
-  </head>
+  <Body>
```

Comment:
Use a lowercase B in <body> (it doesn't really make a difference, but let's go with all lower case tag names as part of our style guide).

Reply:
Thanks for letting me now. Just to make sure, is this the style guide that you're referring to https://github.com/google/styleguide ?

Reply:
I wasn't, but if we want to adopt those guides for our project, that sounds good to me. :)

Reply:
I'm not sure if everyone wants to use these, but this is what I found on the codeU documentation. It's a bit hard to understand some of them so, do you have a link to some other style guide?